Src/sym.c

```diff
@@ -496,6 +496,7 @@ sputtype(char *foo, int m)
 	case INT:	strcpy(foo, "int   "); break;
 	case MTYPE:	strcpy(foo, "mtype "); break;
 	case STRUCT:	strcpy(foo, "struct"); break;
+	case UNION:	strcpy(foo, "union"); break;
```

Comment:
some types have spaces, some not, I wonder why

Reply:
Don't ask me, I don't know

Reply:
heh, ack